README.md

```diff
@@ -12,79 +46,265 @@ cp ./.env.example ./.env
 
 no further editing to the above files is required for running the stack locally.
 
-## Starting the full stack with LS-AAI-mock
+The storage and interfaces stack can be deployed with the use of the provided `docker-compose.yml` file by running
+
+```shell
+docker compose -f docker-compose.yml up -d
+```
+
+from the root of this repo. Please note that in the current form of the compose file, services are configured to work out-of-the-box with the [LS-AAI-mock](https://github.com/GenomicDataInfrastructure/starter-kit-lsaai-mock) service and some configuration of the latter is needed beforehand, see [here](./README.md##Starting-the-full-stack-with-LS-AAI-mock) for step-by-step instructions. The rationale behind this setup is to allow for a seamless transition to an environment with a *live* LS-AAI service as discussed briefly below.
+
+Configuration can be further customized by changing the files listed at the top of the `./.env` file along with the `.env` file itself. Please bear in mind that environment variables take precedence over the `config.yaml` file.
+
+Lastly, this repo includes a `docker-compose-demo.yml` file which deploys a standalone stack along with a demo of the sda services' functionality with test data. Details can be found [here](./README.md##Starting-the-stack-in-standalone-demo-mode).
+
+### Adding TLS to internet facing services
+
+Internet facing services such as `s3inbox`, `download` and `auth`, need to be secured via TLS certification. This can be most conveniently achieved by using [Let's Encrypt](https://letsencrypt.org/getting-started/) as Certificate Authority. Assuming shell access to your web host, a convenient way to set this up is through installing Certbot (or any other ACME client supported by Let's Encrypt). Detailed instructions on setting up Certbot for different systems can be found [here](https://certbot.eff.org/).
+
+For an example of how to set up the `sda-pipeline` with TLS, see [TLS-example/README.md](config/TLS-example/README.md)
+
+## Authentication for users with LS-AAI (mock or alive)
+
+To interact with SDA services, users need to provide [JSON Web Token](https://jwt.io/) (JWT) authorization. Ultimately, tokens can be fetched by [LS-AAI](https://lifescience-ri.eu/ls-login/) upon user login to an OpenID Connect (OIDC) relaying party (RP) service that is [registered with LS-AAI](https://spreg-legacy.aai.elixir-czech.org/). An example of such an RP service is the [sda-auth](https://github.com/neicnordic/sda-auth), which is included in the present stack.
```

Comment:
I think this link https://spreg-legacy.aai.elixir-czech.org/ is outdated. Maybe https://services.aai.lifescience-ri.eu/spreg/ is the correct one. Is that correct @jbygdell ?

Also the link to `sda-auth` should be changed to https://github.com/neicnordic/sensitive-data-archive/tree/main/sda/cmd/auth